### PR TITLE
Create new activity

### PR DESCRIPTION
**Pull request by Jacksonmcguire**
## Is this a fix or a feature?
Feature
## What is the change?
Now, after clicking the log activity button, the timer and button will be replaced with a 
create a new activity button.
Also when the log activity button is pressed the current activity title changes to completed activity.
If the user clicks the create a new activity button, the user will be sent to the main form screen.
## Where should the reviewer start?
After the logActivity function in Main.js and styling for the create new activity button in styles.css
## How should this be tested?
Testing out the buttons functionality on the page

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -178,9 +178,39 @@ function logActivity() {
...
    saveCard();
+   clearTimerSection();
   }
 
-
+  function clearTimerSection() {
+    containerTimer.innerHTML = `<button class="create-new-activity">CREATE A NEW ACTIVITY</button>`;
+    mainTitle.innerText = 'Completed Activity';
+    var createNewActivity = document.querySelector('.create-new-activity');
+    createNewActivity.addEventListener('click', goHome);
```
Comment (elizhahn) [inline main.js]: Should we be utilizing event delegation for this?

Comment (owenhallgren) [inline main.js]: Could we clean this up by doing document.querySelector('.create-new-activity').addEventListener('click', goHome);

**Review by elizhahn — COMMENTED** [review]
I noticed when a new activity is submitted after returning home after hitting the Start new Activity button, the page flips back to that page and skips the timer. We will need to fix that

File: main.js (hunk 1 above)
Comment (Jacksonmcguire) [reply main.js]: For sure, let me know what you think of my updates!

Comment (Jacksonmcguire) [reply main.js]: Ended up finding a way around it by putting the button in the html file to start and attached the event listener directly to it at the top of main js

**Comment by Jacksonmcguire** [comment]
> I noticed when a new activity is submitted after returning home after hitting the Start new Activity button, the page flips back to that page and skips the timer. We will need to fix that

For sure, fixed it temporarily, but going to switch a bug fixing branch to get deeper into it!

File: main.js [hunk 2]
```diff
@@ -60,6 +62,7 @@ function displayTime() {
 function displayTimer() {
   hide(form);
   display(containerTimer);
+  console.log(containerTimer.innerHTML);
```
Comment (elizhahn) [inline main.js]: remove this

File: main.js [hunk 3]
```diff
@@ -125,6 +128,7 @@ function validateForm() {
     // var createdActivity = new Activity(currentIcon.id, inputActivity.value, inputMinutes.value, inputSeconds.value);
     //we will need to call this method on the start button listener instead
     //need to take displaly funcitonality and move it
+    console.log(currentActivity);
```
Comment (elizhahn) [inline main.js]: this one too

File: main.js [hunk 4]
```diff
@@ -125,6 +128,7 @@ function validateForm() {
     // var createdActivity = new Activity(currentIcon.id, inputActivity.value, inputMinutes.value, inputSeconds.value);
     //we will need to call this method on the start button listener instead
     //need to take displaly funcitonality and move it
```
Comment (elizhahn) [inline main.js]: also these haha

File: main.js [hunk 5]
```diff
@@ -178,9 +182,39 @@ function logActivity() {
      }
  }
    saveCard();
+   clearTimerSection();
```
Comment (elizhahn) [inline main.js:180]: not sure why cleartimerSection() is being called inside this saveCare() function

Comment (elizhahn) [reply main.js:180]: Never mind, I read this wrong

**Review by elizhahn — APPROVED** [review]

**Pull request merged**